experimental/packages/exporter-trace-otlp-grpc/src/OTLPTraceExporter.ts

```diff
@@ -42,17 +37,22 @@ export class OTLPTraceExporter
   implements SpanExporter
 {
   constructor(config: OTLPGRPCExporterConfigNode = {}) {
-    super(config);
-    const headers = {
-      ...USER_AGENT,
-      ...baggageUtils.parseKeyPairsIntoRecord(
-        getEnv().OTEL_EXPORTER_OTLP_TRACES_HEADERS
-      ),
-    };
-    this.metadata ||= new Metadata();
+    const headers = baggageUtils.parseKeyPairsIntoRecord(
```

Comment:
It looks like there is quite some duplication here. Can this be moved into the otlp-grpc-exporter-base package or is there some subtle difference that requires them to be separate? It might even be possible to do the merge in the otlp-exporter-base package and just extract it to a metadata object in the otlp-grpc-exporter-base package to further reduce duplication

Reply:
Currently the metadata in the config would be overwritten by env vars. My idea is to merge the env vars and config first, then apply to the base package.